src/repositories/repositories.ts

```diff
@@ -0,0 +1,11 @@
+import { Applicant } from "../models/db";
+import { BaseRepository } from "./baseRepository";
+import { injectable } from "inversify";
+import { Repository } from "typeorm";
+
+@injectable()
+export class ApplicationRepository extends BaseRepository<Applicant> {
```

Comment:
Yeah, makes sense. I thought it would be nice to have ApplicationController, ApplicationService, ApplicationRepository, etc. But since a controller can have multiple services it makes sense to match the name to the DB repository